docker_xylem/utils.py

```diff
@@ -175,15 +184,15 @@ def request(self, url, method='GET', headers={}, data=None, socket=None):
                     raise Exception('HTTPS requested but not supported')
             else:
                 agent = Agent(reactor)
-        
+
         request = agent.request(method, url,
-            Headers(headers),
-            StringProducer(data) if data else None
-        )
+                                Headers(headers),
+                                StringProducer(data) if data else None
+                                )
```

Comment:
Rather than aligning everything with the opening paren of the method call, please move the first param(s) onto the next line and indent everything 4 spaces. For this block, we'd end up with something like:
```python
request = agent.request(
    method, url, Headers(headers),
    StringProducer(data) if data else None)
```

This is very definitely a subjective style choice, but most of our code is written like this and it does have the advantage that the lines with parameters on them don't have giant fields of empty space in front of them. The positioning of the closing paren is also subjective, but I generally prefer to put it at the end of the last parameter to avoid using more vertical space than necessary.